src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -543,6 +543,26 @@
 	IEmoji getEmojiByName(String name);
 
 	/**
+	 * Creates a new emoji.
+	 *
+	 * @param name The name, <b>without colons</b> of length 2-32 characters.
+	 * @param image The image of the emoji.
+	 * @param roles The roles for which this emoji will be whitelisted
```

Comment:
Maybe clarify that this will only work for whitelisted guilds to avoid future confusion/questions.